prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1686,7 +1686,7 @@ static inline int rxr_ep_send_queued_pkts(struct rxr_ep *ep,
 {
 	struct dlist_entry *tmp;
 	struct rxr_pkt_entry *pkt_entry;
-	int ret;
+	ssize_t ret;
```

Comment:
Thanks!

The return type of `rxr_ep_send_queued_pkts` should be `ssize_t`, too. I think.

Reply:
Right, updated.